tests/test_execution_compute_plan.py

```diff
@@ -332,7 +332,7 @@ def test_compute_plan_aggregate_composite_traintuples(global_execution_env):
     previous_composite_traintuple_specs = []
 
     cp_spec = factory.create_compute_plan(
-        cleanModels=True
```

Comment:
Maybe we could add a dedicated test for this feature. 
IMO, it's not straightforward to understand what's the purpose of setting this option to true and how to test it. 

We could maybe implement the following scenario:

1. create a small CP with clean_models enabled
2. wait for the CP to be executed
3. create a new tuple based on an intermediary model
4. check the tuple failed

If we don't want to create a new test, we could just add steps 3 and 4 to this test.

Reply:
Maybe we could also check the `shared_path` if available to make sure objects were removed physically.

Reply:
What do you mean by the `shared_path` ?

Reply:
For the local test environment (setup through skaffold), there is a `shared_path` pointing to the servermedias folder (see `local_values.json`) that is accessible both by the tests and the platform. This was done to test adding data samples using a local path instead of an upload.

My thinking was that this path probably also contains the generated models and that we could check if physically the models were deleted or not.

Reply:
Checking model existence can be an issue because we cannot be sure that we will have an access to the models volume in the future !